webapp/packages/plugin-data-viewer/src/locales/it.ts

```diff
@@ -39,6 +46,7 @@ export default [
   ['data_viewer_refresh_result_set', 'Refresh result set'],
   ['data_viewer_total_count_tooltip', 'Get total count'],
   ['data_viewer_total_count_failed', 'Failed to get total count'],
+  ['data_viewer_total_count_cancel_failed_title', 'Failed to cancel getting of total count'],
```

Comment:
I am not sure that we should create new message token as "Task was cancelled" is throw by the task itself